rocky/katalogus/client.py

```diff
@@ -129,7 +129,7 @@ def __init__(self, error: httpx.HTTPStatusError):
 
 
 class KATalogusClientV1:
-    def __init__(self, base_uri: str, organization: str):
+    def __init__(self, base_uri: str, organization: str | None):
         self.session = httpx.Client(base_url=base_uri)
         self.organization = valid_organization_code(organization) if organization else organization
         self.organization_uri = f"/v1/organisations/{organization}"
```

Comment:
this url would be malformed if no orga is present

Reply:
Also see https://github.com/minvws/nl-kat-coordination/issues/3697, which I will pick up right away